frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -132,7 +132,7 @@ class DashboardGrid extends Component {
     });
 
     if (changes.length > 0) {
-      setMultipleDashCardAttributes(changes);
+      setMultipleDashCardAttributes({ dashcards: changes });
```

Comment:
This was incorrect before. The `setMultipleDashCardAttributes` is an action creator made by the `createAction` method from `redux-actions`, so its argument should be an object that is the payload for the action. This action takes in a payload object with a single attribute `dashcards` (see [`dashboard/reducers.js`](https://github.com/metabase/metabase/blob/e562f3e091b8f6fdfee592605f7e6da9b4d0c61c/frontend/src/metabase/dashboard/reducers.js#L169)).